tests/cicd/TLKJenkinsfile

```diff
@@ -0,0 +1,93 @@
+pipeline {
+    agent {
+        label 'sr613'
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                script {
+                    def labels = ['sr613']
+                    def builders = [: ]
+                    for (x in labels) {
+                        def label = x
+                        builders[label] = {
+                            node(label) {
+                                cleanWs()
+                                checkout scm
+                                sh 'cd Dockerfile-ubuntu18.04 && docker build -t aidk-pytorch110 . -f DockerfilePytorch110 && cd .. && yes | docker container prune && yes | docker image prune'
+                            }
+                        }
+                    }
+                    parallel builders
+                }
+            }
+        }
+        stage('Run transfer_learning_unittest') {
+            agent {
+                label 'sr613'
+            }
+            steps {
+                sh 'docker run --rm --name test-tlk-singlenode --privileged --network host --device=/dev/dri --shm-size="2g"  -v /mnt/DP_disk1/tlk:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app -w /home/vmagent/app/test/cicd aidk-pytorch110 /bin/bash -c ". /home/vmagent/app/tests/cicd/jenkins_transfer_learning_unittest.sh"'
+            }
+        }
+        stage('Run transfer_learning_DDP_baseline_test') {
+            agent {
+               label 'sr613'
+            }
+            steps {
+               sh 'docker run -t -d --name test-tlk-multinode-baseline-r1 --privileged --network host --device=/dev/dri -v /mnt/DP_disk1/tlk:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app -w /home/vmagent/app/AIDK/TransferLearningKit/src aidk-pytorch110 /bin/bash -c ". /home/vmagent/app/tests/cicd/jenkins_transfer_learning_basic_test_ddp.sh"'
```

Comment:
1. Since ModelAdapter runs on PyTorch1.12 env, this image need to change to e2eaiok-pytorch112
2. Since the project name is e2eaiok, need to change aidk to e2eaiok.

Reply:
Updated